app/models/pipeline_run.rb

```diff
@@ -400,7 +400,7 @@ def db_load_taxon_counts
   end
 
   def db_load_refined_taxon_counts
-    output_json_s3_path = "#{postprocess_output_s3_path}/#{REFINED_TAXON_COUNTS_JSON_NAME}"
+    output_json_s3_path = s3_file_for("refined_taxon_counts")
```

Comment:
refined_taxon_counts doesn't seem to exist in `s3_file_for` any more. Did you mean `taxon_counts`?

Reply:
Ah I see. I guess this loader doesn't get called any more.

Can you add some comments with `TODOs` just so others don't get confused?